tests/quality/test_quality.py

```diff
@@ -0,0 +1,143 @@
+import os
+
+import numpy as np
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import cross_val_score
+
+from rdt import HyperTransformer
+from rdt.transformers import NumericalTransformer, get_transformers_by_type
+from tests.quality.utils import download_single_table_dataset
+
+THRESHOLD = 0.2
```

Comment:
I lowered this because there were no tests that had higher correlations than 0.4 for datetime data types